app/lib/l10n/app_en.arb

```diff
@@ -1,32 +1,20 @@
 {
-  "app_feature_coming_soon": "Have a little patience, this feature is coming soon!",
-  
   "auth_choose_lab": "Please select your lab",
   "auth_sign_in": "Sign in",
   "auth_success": "Successfully imported data",
   
-  "err_could_not_launch": "Could not launch {url}",
-  "@err_could_not_launch": {
-    "description": "Error message when (auth-related) url cannot be launched.",
-    "placeholders": {
-      "url": {
-        "type": "String",
-        "example": "https://www.google.de"
-      }
-    }
-  },
   "err_could_not_retrieve_access_token": "An unexpected error occurred while retrieving the access token",
   "err_fetch_user_data_failed": "An unexpected error occurred while fetching your genomic data",
   "err_generic": "Error!",
   
-  "faq_page_description": "On this page we will answer common questions regarding PGx",
+  "faq_page_description": "Here you can find answers to common questions about PGx",
   "faq_pharmacogenomics": "Pharmacogenomics",
 
   "general_appName": "PharMe",
   "general_continue": "Continue",
   "general_retry": "Retry",
   
-  "medications_page_disclaimer": "This assessment is only based on your genomic data. It does not consider other factors such as weight, age and pre-existing conditions.",
+  "medications_page_disclaimer": "This assessment is ONLY based on your genetics. Important factors like weight, age and pre-existing conditions are not considered.",
```

Comment:
This all caps seems a little childish IMO. Also would change second sentence slightly + add oxford comma:

```suggestion
  "medications_page_disclaimer": "This assessment is only based on your genetics. Important factors such as weight, age, or pre-existing conditions are not considered.",
```